### PR TITLE
1166 Limit file size

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/views/limitFileSize.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const limitFileSize = (e, callback) => {
+  // 500MB limit
+  if (e.target.files[0].size >= 500 * 1024 * 1024) {
```
Comment (kkoskelin) [inline web-client/src/views/limitFileSize.js]: I wouldn't mind if there were
```
const MB = 1024 * 1024;
export const MAX_SIZE = 500 * MB;
```
but I'm a sucker for syntactic sugar.

File: web-client/src/views/limitFileSize.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    alert('Your file must be less than 500 MB.');
+    e.target.value = null;
+  } else {
+    callback();
```
Comment (kkoskelin) [inline web-client/src/views/limitFileSize.js:15]: If we used a promise-based pattern, you'd be able to `limitFileSize(e).then(updatePetitionValueSequence).catch(someKindOfWarningSequence)`

**Review by kkoskelin — APPROVED** [review]
A good place to start.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/views/limitFileSize.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/views/limitFileSize.js]: good idea, updated

**Pull request merged**